src/Sulu/Bundle/MediaBundle/Controller/CollectionController.php

```diff
@@ -368,14 +369,16 @@ protected function saveEntity($id, Request $request)
             $collectionManager = $this->getCollectionManager();
             $data = $this->getData($request);
             $data['id'] = $id;
-            $data['locale'] = $this->getLocale($request);
+            $data['locale'] = $this->getRequestParameter($request, 'locale', true);
             $collection = $collectionManager->save($data, $this->getUser()->getId());
 
             $view = $this->view($collection, 200);
         } catch (CollectionNotFoundException $cnf) {
             $view = $this->view($cnf->toArray(), 404);
         } catch (MediaException $me) {
             $view = $this->view($me->toArray(), 400);
+        } catch (MissingParameterException $mp) {
```

Comment:
We call the exception variables in the `catch` block `$e`, please also fix for the other catch statements.